src/main/java/org/spongepowered/api/event/command/CommandProcessEvent.java

```diff
@@ -63,28 +94,4 @@
      */
     String getArguments();
 
-    /**
-     * Set the arguments as a string.
-     *
-     * <p>For example, if the message was {@code /example bob 3 -f}, then
-     * the arguments would be {@code bob 3 -f}.</p>
-     *
-     * @param arguments The arguments
-     */
-    void setArguments(String arguments);
-
-    /**
-     * The result of the command.
-     *
-     * @return The result of the command
-     */
-    CommandResult getResult();
-
-    /**
-     * Sets the result of the command.
-     *
-     * @param result The result of the command
-     */
-    void setResult(CommandResult result);
```

Comment:
The removal of setResult from the Pre event, means that plugins which provide command redirection / rewriting will be unable to return the result of the original processed command, when the event is cancelled.

Whether this is a good or bad thing however I can't decide.

You could argue that instead of cancelling, you should just rewrite the redirected command into the original and let it continue if you actually need the CommandResult.

But without setting a result, cancelling would have to make an assumption about the appropriate CommandResult to return, as https://github.com/SpongePowered/SpongeAPI/blob/11a6c07c829b1306a24b4792b6f5dc0dd0556956/src/main/java/org/spongepowered/api/command/CommandManager.java#L155 does not return an Optional<CommandResult>